### ISSUE TITLE
Allow Custom Location for screenshot folder

### ISSUE THREAD

**Issue by TavoreParan** [runelite/runelite#1324]
Currently you can't change the location of where your screenshots are saved, which is sooner or later is going to make it an issue especially if its saving to an SSD. So just being able to change location would be amazing, and I don't think its a hard thing to add either.

**Comment by sycp001** [comment]
you added this functionality but how do you actually go about changing the location?

**Comment by deathbeam** [comment]
You can already do this with symlinks on windows: https://www.techrepublic.com/article/how-to-take-advantage-of-symbolic-links-in-window-10/

**Comment by itchysinus** [comment]
> 
> 
> You can already do this with symlinks on windows: https://www.techrepublic.com/article/how-to-take-advantage-of-symbolic-links-in-window-10/

How did you manage this?  Mklink creates the source folder and it can not already exist.

**Comment by Nightfirecat** [comment]
You'd remove (or rename) the existing screenshots directory, create a symlink, then move your existing screenshots to the new directory.

**Comment by megumeat** [comment]
Can there be a guide or something? Or can this request be reopened? I really don't understand how to do a symlink or whatever.

**Comment by Jerrk** [comment]
seems odd to not give us the option to change the directory and just tell us to make a symlink instead which sort of gatekeeps the functionality behind some sort of technological know-how.

**Comment by Adam-** [comment]
I think the original concern was if people play on multiple pcs the screenshot location would be synced between pcs which wouldn't work correctly. And we don't have a way to do local-only config settings.

**Comment by Jerrk** [comment]
ah yes i can see how that would be an issue if you make use of the account feature.

Since i already run multiple runelite windows with separate .properties files through the clientargs option i thought it might be possible to do something like that in your runelite shortcut for the screenshots location.

though not sure how big the desire for this feature is when theres only been 3 topics all posted over 3 years ago asking for it. dev-time is probably better spent elsewhere if the symlink option is viable.

**Comment by DaMage93** [comment]
So here's how I made a symlink from the usual c:\Users\{user}\.runelite directory that houses all of the settings including the screenshot folder.

First make a copy of the .runelite directory as a backup in case things go bad.

Now we have a ".runelite" and a ".runelite - Copy" in c:\Users\{user} directory

Move the .runelite directory to where you want it to be. Mine is on an external drive E:\Runelite. So now I have the runelite directory in E:\Runelite\.runelite

Type "cmd" for "Command Prompt" in the windows search bar at the bottom of the screen and right click "Run as Administrator". Click Yes.

Now run the command:

`mklink /D c:\Users\{user}\.runelite e:\Runelite\.runelite`

mklink is the command to make symbolic links. /D is specifying this will be a directory and not a file.

Just replace where your original runelite was (mine was c:\Users\{user}\.runelite) and where you want the runelite to be now (mine was e:\Runelite\.runelite).

This should work. I just opened runelite without anything going bad, but I will keep the copy for a while to ensure that things are going well. Next time I get a level up, I'll make sure that the new screenshot will go in the new location.
